packages/browser/src/index.ts

```diff
@@ -29,6 +29,10 @@ export { INTEGRATIONS as Integrations };
 export { Replay } from '@sentry/replay';
 // __ROLLUP_EXCLUDE_REPLAY_FROM_BUNDLES_END__
 
+// __ROLLUP_EXCLUDE_BROWSER_TRACING_FROM_BUNDLES_BEGIN__
+export { BrowserTracing } from '@sentry-internal/tracing';
```

Comment:
Hmm, should we export this from `src/integrations/index.ts` instead? As this is where you'd currently get this from e.g. in CDN bundles. Also where I hooked this in for https://github.com/getsentry/sentry-javascript/pull/7479.

Reply:
It might make more sense to put it in `src/integrations/index.ts` since it IS an integration. However, the `Replay` integration is exported just above this and I thought it might be better to keep all the `__ROLLUP_EXCLUDE` wrapped exports together near the comments describing why they shouldn't be deleted.

Reply:
Hmm, good question... So I think either way we have to make sure it is available in both places, as that's where we currently have it. So both of these work as of today for CDN bundles:

```
new Sentry.BrowserTracing();
new Sentry.Integrations.BrowserTracing();
```

So TLDR I guess it's fine to export it from the root, but we need to make sure that it is _also_ on `Sentry.Integrations` in CDN bundles.

Reply:
We can do this in a follow up given the bundle is already being generated from https://github.com/getsentry/sentry-javascript/blob/develop/packages/tracing/src/index.bundle.base.ts - so I think we should be fine.